content/docs/user-guide/project-structure/pipelines-files.md

```diff
@@ -156,6 +160,49 @@ stages:
 DVC will track simple param values (numbers, strings, etc.) used in `${}` (they
 will be listed by `dvc params diff`).
 
+### Dict Unpacking
+
+Only inside the `cmd` entries, you can also reference a dictionary inside `${}`
```

Comment:
I forget what happens if this is tried outside of `cmd`?

Reply:
Something like:

```
ResolveError("failed to parse 'stages.build.outs' in 'dvc.yaml':\nCannot interpolate data of type 'dict'")
```